Extensions/blacklist.js

```diff
@@ -209,7 +210,8 @@ XKit.extensions.blacklist = new Object({
 
 		}
 
-		if ($(postSel).length > 0) {
+		//running on every react page until xkit handles tumblr's soft refresh
```

Comment:
i think it's clear enough why the `XKit.page.react1 check is here without the comment
```suggestion
```